source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -192,6 +192,116 @@ the Query API:
                                       then: "extreme temperature",
                                       else: "no advisory" } } } }
 
+Conditional Expression Operations
```

Comment:
S: I believe we are omitting "Expression" in these titles. Also applies for set section below